bool request_tag  # ID must be echoed back in SampleResult

